client/src/components/ExportCsv.tsx

```diff
@@ -0,0 +1,85 @@
+import {formatIsoDate} from 'date/utils';
+import {ExportToCsv} from 'export-to-csv';
+
+const sum = (acc: number, item: number) => acc + item;
+
+function csvExporter(users: any) {
+  let data = users.map((row: any) => {
+    return {
+      Username: row.displayName,
+      'Change Requests': row.repoTotals
+        .map((repo: any) => repo.authoredTotals.changesCreated)
```

Comment:
There should be no usages of `any`, everything in the project is strongly typed. Let's continue keeping it that way!

Reply:
You're right, I am new to TypeScript and I am still learning it. I would appreciate it if you let me know what types should I use instead of `any`

Reply:
Of course! So if you use VS Code (which I'd highly recommend - fantastic editor), you can hover over the types that already exist. This is what you're passing in:
<img width="402" alt="Screenshot 2022-04-23 at 17 01 06" src="https://user-images.githubusercontent.com/3183122/164911549-04b8b16d-be95-4c35-a31d-a3a916460f49.png">
 
In addition, you'd want to keep the types immutable to tell the caller they don't have to worry about modifications, and to prevent bugs. So you'd want `users: ReadonlyArray<UserResult>` in the Props type for the button, and you can probably just delete the `:any` type qualifiers once you've done that; the compiler will infer the right types for you.

Reply:
I didn't quite get the error. Can you please tell me what it says? 
![image](https://user-images.githubusercontent.com/35958006/164916009-8ba36622-da7e-4ad4-84ea-734d25b12a01.png)

Reply:
Sure thing - what the compiler is trying to say that you're trying to pass in a read-only array to a function whose first argument is not an array. that is, `ReadonlyArray<UserResult>  !== UserResult`. I think that if you change the `csvExporter` function's arg to `ReadonlyArray<UserResult>`, it'll work.

Reply:
Yeah I'm getting the above error after changing the `csvExporter` function's arg to `ReadonlyArray<UserResult>` as 
```
function csvExporter(users: ReadonlyArray<UserResult>) {
  const data = users.map((row) => {
    return {
            .........
```
![image](https://user-images.githubusercontent.com/35958006/164973484-e6255bb0-4bf7-490a-8d35-1b0f89146847.png)